src/lib/featureFlags.ts

```diff
@@ -0,0 +1,43 @@
+import config from "config"
+import { Unleash, initialize } from "unleash-client"
+import { info, error } from "./loggers"
+
+const { UNLEASH_API, UNLEASH_APP_NAME, UNLEASH_SERVER_KEY } = config
+
+/**
+ * Feature flags are defined within Unleash.
+ * @see https://tools.artsy.net/feature-flags
+ */
+const FEATURE_FLAGS_LIST = ["diamond_blurhash-enabled-globally"] as const
+
+type FeatureFlag = typeof FEATURE_FLAGS_LIST[number]
+
+let unleashClient: Unleash
+
+export function initilizeFeatureFlags() {
+  unleashClient = initialize({
+    url: UNLEASH_API,
+    appName: UNLEASH_APP_NAME,
+    customHeaders: { Authorization: UNLEASH_SERVER_KEY },
+  })
```

Comment:
Do we run the risk of instantiating multiple clients unless we guard against it? If `src/index.ts` is run more than once, than I think this might be an issue.

Reply:
Index.ts isn't run more than once (unless in dev mode, and we've hot reloaded a change), so all ok!